cpufeatures/Cargo.toml

```diff
@@ -19,3 +19,6 @@ libc = "0.2.68"
 
 [target.'cfg(all(target_arch = "aarch64", target_os = "linux"))'.dependencies]
 libc = "0.2.68"
+
+[target.aarch64-linux-android.dependencies]
+libc = "0.2.68"
```

Comment:
Kind of confused why the above doesn't suffice for this. Is `target_arch` or `target_os` actually different from the above on Android?

Reply:
Yeah, that's the only reason the existing implementation *didn't* work.